common-design-patterns/language-toggle.md

```diff
@@ -0,0 +1,192 @@
+      <p class="mrgn-tp-lg">Standard header of a French Canada.ca page with a highlight of the linked abbreviation EN in the top-right corner</p>
+    </details>
+  </figure>
+</div>
+<h2 id="implementation">How to implement</h2>
+<p>Find working examples for implementing the language toggle.</p>
+<h3>GCweb (WET) theme implementation reference</h3>
+<p>The implementation reference includes how to configure each element of the header.</p>
+<ul>
+  <li><a href="https://wet-boew.github.io/GCWeb/sites/header/header-docs-en.html">GCWeb (WET) header documentation</a></li>
+</ul>
+<h3>Implementations</h3>
+<p>Determine what best suits the type of page you're creating.</p>
+<div class="row">
+  <div class="col-md-8">
+    <div class="wb-tabs mrgn-tp-lg">
+      <div class="tabpanels">
+        <details id="004" open="open">
+          <summary><strong>GC-AEM</strong></summary>
+          <p class="mrgn-tp-lg">For the Government of Canada Adobe Experience Manager (AEM):</p>
+          <ul>
+            <li><a href="https://www.gcpedia.gc.ca/wiki/AEM_GC-specific_Documentation_6.5">AEM/Managed Web Service documentation (GCPedia link - only available on the Government of Canada network)</a></li>
+          </ul>
+        </details>
+        <details id="005">
+          <summary><strong>CDTS</strong></summary>
+          <p class="mrgn-tp-lg">For the Centrally Deployed Templates Solution (CDTS):</p>
+          <ul>
+            <li><a href="https://cenw-wscoe.github.io/sgdc-cdts/docs/index-en.html">CDTS documentation</a></li>
+          </ul>
+        </details>
+        <details id="006">
+          <summary><strong>Drupal WxT</strong></summary>
+          <p class="mrgn-tp-lg">For Drupal WxT:</p>
+          <ul>
+            <li><a href="https://drupalwxt.github.io/en/">Drupal WxT documentation</a></li>
+          </ul>
+        </details>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="cnt-wdth-lmtd">
+  <h2 id="research">Research and rationale</h2>
+  <p>Consult policy rationale.</p>
+  <h3>Policy rationale</h3>
+  <p>As part of the global header, the language toggle is a mandatory element under the Content and Information Architecture Specification.</p>
+  <ul>
+    <li><a href="https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-information-architecture-specification/mandatory-elements.html">Mandatory elements of the design system</a></li>
+  </ul>
+  <p>All Government of Canada communications must be available in both official languages.</p>
+  <ul>
+    <li><a href="https://www.tbs-sct.canada.ca/pol/doc-eng.aspx?id=26160">Policy on Official Languages</a></li>
+  </ul>
+  <h2 id="changes">Latest changes</h2>
+  <dl class="dl-horizontal">
+    <dt>
+      <time datetime="2023-MM-DD" class="link-muted">2023-MM-DD</time>
+    </dt>
+    <dd>Updated the guidance to include content and design specifications, visual examples and implementation guidance</dd>
```

Comment:
```suggestion
    <dd>Updated the guidance to include advice on what to avoid, content and design specifications, visual examples, implementation guidance, and policy rationale</dd>
```